.kitchen.yml

```diff
@@ -4,6 +4,9 @@ driver:
   customize:
     memory: 1024
 
+provisioner:
+  data_path: test/shared  # directory containing shared helpers for integration tests
```

Comment:
This should not be needed - we already support shared directories via the `test/integration/helpers` subdirectory, so this should also simplify the path handling.

If there's a common spec helper, then let's move all the common `require`s and `set path` into there.